src/main/java/Leas_Liudmila/files/GradeBook.java

```diff
@@ -0,0 +1,108 @@
+package Leas_Liudmila.files;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class GradeBook {
+
+    private String fileName = "files/gradesbook.csv";
+
+    public void saveStudent(Student studentToSave) {
+
+        String studentInCsv = convertStudentToCsvLine(studentToSave);
+
+        try {
+
+            File path = new File(fileName);
+            if(path.exists() == false) {
+                path.createNewFile();
+            }
+
+            FileWriter fw = new FileWriter(fileName, true);
+
+            BufferedWriter bw = new BufferedWriter(fw);
+            bw.write(studentInCsv);
+            bw.newLine();
+
+            bw.close();
+            fw.close();
+        } catch (IOException e) {
+            System.out.println("Unable to save in CSV: " + studentInCsv);
+        }
+    }
+
+    public ArrayList<Student> loadStudents() throws IOException {
+
+        File path = new File(fileName);
+        if(path.exists() == false) {
+            throw new IOException("File not found or not exists: " + fileName);
+        }
+
+        ArrayList<Student> result = new ArrayList<Student>();
+
+        Scanner read = new Scanner(path);
+
+        while (read.hasNextLine()) {
+            String csvStudent = read.nextLine();
+
+            if(csvStudent.length() > 1) {
```

Comment:
in order to skip first line with column names you can before loop make read.nextLine() once

Reply:
This verification skips empty lines, not column names